doc/plans/reflectometry/detector_mapping_alignment.md

```diff
@@ -125,11 +125,11 @@ from bluesky.utils import Msg
 from ibex_bluesky_core.utils import get_pv_prefix
 from ibex_bluesky_core.devices.block import block_rw
 from ibex_bluesky_core.devices.simpledae import SimpleDae
-from ibex_bluesky_core.devices.simpledae.controllers import PeriodPerPointController
-from ibex_bluesky_core.devices.simpledae.reducers import (
+from ibex_bluesky_core.devices.simpledae._controllers import PeriodPerPointController
+from ibex_bluesky_core.devices.simpledae._reducers import (
     PeriodSpecIntegralsReducer,
 )
-from ibex_bluesky_core.devices.simpledae.waiters import PeriodGoodFramesWaiter
+from ibex_bluesky_core.devices.simpledae._waiters import PeriodGoodFramesWaiter
```

Comment:
We can import these from top-level of `simpledae` now?

Reply:
Did we discuss this and couldn't resolve it? We should need `_` imports in the docs, if we do then it's a sign our API is wrong...